resources/META-INF/plugin.xml

```diff
@@ -16,7 +16,7 @@
   <vendor>Twitter, Inc.</vendor>
 
   <!--if you are changing since-build don't forget to change it in .travis.yml file as well-->
-  <idea-version since-build="172.1657" until-build="173.0"/>
+  <idea-version since-build="172.*" until-build="173.0"/>
```

Comment:
ah looks like `since-build` cannot be wildcard, `until-build` can, so
```
  <idea-version since-build="172.0" until-build="172.*"/>
```
should do.